index.html

```diff
@@ -351,15 +408,22 @@ <h3>
             If the domain contains a port percent decode the colon.
           </li>
           <li>
-            Generate an HTTPS URL to the expected location of the DID document
+            Generate an HTTPS URL to the expected location of the DID Resolution Document
             by prepending <code>https://</code>.
           </li>
           <li>
             If no path has been specified in the URL, append
             <code>/.well-known</code>.
           </li>
           <li>
-            Append <code>/did.json</code> to complete the URL.
+            Map the DID Document Option's accept property to a file extension:
```

Comment:
If we are going to do content negotiation, we should follow the HTTP standard: https://www.rfc-editor.org/rfc/rfc7231#section-5.3

Reply:
> If we are going to do content negotiation, we should follow the HTTP standard

Strong +1 on following HTTP. DID core is very consistent with HTTP content negotiation.

https://www.w3.org/TR/did-core/#did-resolution-options

https://www.w3.org/TR/did-core/#did-resolution-metadata